hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -66,68 +61,140 @@
                 once the event interrupt pin is asserted
             '''
       milestone: V2
-      tests: ["spi_host_error_event_intr"]
```

Comment:
So are these removed directed tests now effectively handled as covergroups in the new plan?